src/BayesFilters/include/BayesFilters/LTIMeasurementModel.h

```diff
@@ -0,0 +1,32 @@
+#ifndef LTIMEMEASUREMENTMODEL_H
+#define LTIMEMEASUREMENTMODEL_H
+
+#include <BayesFilters/LinearMeasurementModel.h>
+
+#include <Eigen/Dense>
+
+namespace bfl {
+    class LTIMeasurementModel;
+}
+
+
+class bfl::LTIMeasurementModel : public bfl::LinearMeasurementModel
+{
+public:
+    LTIMeasurementModel(const Eigen::MatrixXf& measurement_matrix, const Eigen::MatrixXf& noise_covariance_matrix);
```

Comment:
I generally prefer to have `const Eigen::Ref<const Eigen::MatrixXf>&` so that user can also pass blocks and other expressions. Is there a particular reason not ot do so?

Reply:
Done!